server/monitor_test.go

```diff
@@ -1313,18 +1313,18 @@ func TestConcurrentMonitoring(t *testing.T) {
 			for i := 0; i < 150; i++ {
 				resp, err := http.Get(url + endpoint)
 				if err != nil {
-					t.Fatalf("Expected no error: Got %v\n", err)
```

Comment:
Thanks @derekcollison @kozlovic I was trying to address the `the goroutine calls T.Fatalf, which must be called in the same goroutine as the test` error which shows up, but this is the wrong approach...
I have a similar PR up for the client but will leave it on pending/close as it shows the same issue https://github.com/nats-io/nats/pull/237